esy/Config.ml

```diff
@@ -31,7 +31,7 @@ let create
       match prefixPath with
       | Some v -> return v
       | None ->
-        let%bind home = Bos.OS.Dir.user() in
+        let%bind home = Path.ofString(System.Environment.homeDir ()) in
```

Comment:
This should be just `let home = Path.homePath in` if we do changes proposed above.